src/WWTelematics.TenantService/Validators/TenantValidator.cs

```
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using FluentValidation;
+using WWTelematics.TenantService.Models;
+
+namespace WWTelematics.TenantService.Validators
+{
+    /// <summary>
+    /// Tenant Validator.
+    /// </summary>
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Naming", "CA1710:Identifiers should have correct suffix", Justification = "Validators implement IEnumerable to allow looking at rules, but are not fundamentally collections.")]
+    public class TenantValidator : AbstractValidator<Tenant>
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TenantValidator"/> class.
+        /// </summary>
+        public TenantValidator()
```

LicenseCount is a required parameter and must be checked.

Currently, passing licenseCount = -1 works (should throw a validation error).
Not passing licenseCount works, creating a Tenant with licenseCount=0; this should throw a validation error.

---

Added validation for "licenseCount " property.
Also marked "licenseCount" property IsRequired in TenantContext file.